tests/logging_callback_tests/test_logging_redaction_e2e_test.py

```diff
@@ -118,3 +118,36 @@ async def test_global_redaction_off_with_dynamic_params(turn_off_message_logging
             == "hello"
         )
         assert standard_logging_payload["messages"][0]["content"] == "hi"
+
+
+@pytest.mark.asyncio
+async def test_redaction_responses_api():
```

Comment:
wasn't the issue with streaming? I think you need a stream + non stream test

Reply:
done